index.html

```diff
@@ -49,7 +49,147 @@ <h2>I am a Software Developer</h2>
+                <h2>Professional Art <br> Printing Data</h2>
+                <p>A daily selection of privately personalized needs; no accounts or sign-ups required. has been the
+                    industry's standard</p>
+                <ul class="languages">
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+                <button class="button" type="button">See Project</button>
+            </article>
+            <article class="project">
+                <h2>Professional Art <br> Printing Data</h2>
+                <p>A daily selection of privately personalized needs; no accounts or sign-ups required. has been the
+                    industry's standard</p>
+                <ul class="languages">
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+                <button class="button" type="button">See Project</button>
+            </article>
+        </section>
+        <section class="about-me">
+            <h2>About Me</h2>
+            <p>Hello I'm a software developer! I can help you build a product, feature or website. Look through some of
+                my work and experience. If you like what you see and have a project you need coded, don't hesitate to
+                contact me.</p>
+            <button class="resume" type="button">Get my resume</button>
+            <div class="lists">
+                <hr id="second-line">
+                <ul>
+                    <li class="bold">
+                        <img src="images/Rectangle 55.png" alt="rhombus">
+                        Language
+                    </li>
+                    <ul class="simple">
+                        <li>Javascript</li>
+                        <li>Ruby</li>
+                        <li>Html</li>
+                        <li>Css</li>
+                    </ul>
+
+                </ul>
+                <ul>
+                    <li class="bold">
+                        <img src="images/Rectangle 56.png" alt="rectangle">
+                        Frameworks
+                    </li>
+
+                    <ul class="simple">
+                        <li>Bootstrap</li>
+                        <li>Ruby on Rails</li>
+                        <li>RSPec</li>
+                        <li>CapyBara</li>
+                        <li>Selenium</li>
+                    </ul>
+
+                </ul>
+                <ul>
+                    <li class="bold">
+                        <img src="images/Rectangle 57.png" alt="circle">
+                        Skills
+                    </li>
+                    <ul class="simple">
+                        <li>Codekit</li>
+                        <li>Githubt</li>
+                        <li>Codepen</li>
+                        <li>Gitlab</li>
+                        <li>Terminal</li>
+                    </ul>
+
+                </ul>
```

Comment:
- [Optional] Great job for adding technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `<ul>` and each of them a `<li>`👍👍,  I think it is better to make each list Items in `<a>` tag to, they may have the corresponding reference link.

Reply:
Will keep it in mind when finishing the project! Thanks!